MEMBERSHIP.md

```diff
@@ -5,8 +5,24 @@ to become a member. At the moment, the only privilege membership brings is being
 to vote for the project leader. And, of course, the glory of having your name appended
 to this file.
 
+## Applying
+
 To apply for membership, send a pull request to this file, adding your name and
-GitHub ID. 
+GitHub ID.
+
+We also like to encourage you to use the pull request's comment to introduce yourself.
+For example what got you interested in SFOSC, or some related work to talk about.
+This is optional though, please share only as much as you would like to.
```

Comment:
Proposal:
Please mention why you became interested in SFOSC and share some related work that you have undertaken.
This is optional though, please disclose only as much as you feel comfortable sharing.

Reply:
Thanks :] I like your suggestion better as well.